src/Poll.ts

```diff
@@ -163,7 +173,7 @@ export class Poll {
         const users: string[] = votes[key].split(",");
         users.splice(0, 1);
         // Don"t bother with empty votes
-        if (users.length === 0) return null;
+        if (users.length === 0) return Static.buildSectionBlock(`*0* ${key} »`);
```

Comment:
You can get rid of this if check altogether and let it fall through to the return on line 179.

Reply:
Fixed. 😃